prow/cmd/gerrit/main.go

```diff
@@ -72,10 +78,91 @@ func gatherOptions() options {
 	flag.StringVar(&o.cookiefilePath, "cookiefile", "", "Path to git http.cookiefile, leave empty for anonymous")
 	flag.Var(&o.projects, "gerrit-projects", "Set of gerrit repos to monitor on a host example: --gerrit-host=https://android.googlesource.com=platform/build,toolchain/llvm, repeat flag for each host")
 	flag.StringVar(&o.lastSyncFallback, "last-sync-fallback", "", "Path to persistent volume to load the last sync time")
```

Comment:
Let's make a separate flag for the gcs path, and deprecate the old one?

Reply:
Why? The whole point is that you can specify a `/local/path` or `gs://remote/path` and it will do the right thing.

Reply:
hummm... sure, update the help message say it can support both then

Reply:
Done